app/server/api/images/upload.post.ts

```diff
@@ -0,0 +1,9 @@
+export default eventHandler(async (event) => {
+  return hubBlob().handleUpload(event, {
```

Comment:
cloudflare creds are not inputted so hubBlob() is proxied as a object in local storage